server/packetio_test.go

```diff
@@ -57,7 +57,8 @@ func TestPacketIORead(t *testing.T) {
 	// Test read one packet
 	brc := newBufferedReadConn(&bytesConn{inBuffer})
 	pkt := newPacketIO(brc)
-	bytes, err := pkt.readPacket()
+	var defMaxAllowedPacket uint64 = 67108864
```

Comment:
Please update it according to https://github.com/pingcap/tidb/pull/33651#discussion_r841070585